corehq/apps/hqwebapp/tasks.py

```diff
@@ -259,7 +259,7 @@ def clear_expired_oauth_tokens():
     call_command('cleartokens')
 
 
-@periodic_task(run_every=crontab(day_of_week=1))
+@periodic_task(run_every=crontab(minute=0, hour=1, day_of_week='mon'))
```

Comment:
did the day_of_week=1 sent the email at a wrong time?

Reply:
It seems like it. Apparently there was emails sent every hour or so, but this new configuration seems to have fixed it. It's weird.